docker-deploy.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/sh
 
+error() {
+  echo $1
+  exit 1
+}
+
+test -z "${VERSION}"         && error "VERSION is required"
+test -z "${DOCKER_USERNAME}" && error "DOCKER_USERNAME is required"
+test -z "${DOCKER_PASSWORD}" && error "DOCKER_PASSWORD is required"
```

Comment:
You don't need this because `docker login` should also not be in here

Reply:
they had it, so I can remove it if you prefer